rudder-webapp/SOURCES/rudder-webapp-postinst

```diff
@@ -197,7 +197,7 @@ echo "Done"
 
 # Restart the webapp
 echo -n "INFO: Make sure Rudder webapp is started..."
-systemctl restart rudder-jetty >> ${LOG_FILE}
+systemctl restart rudder-jetty || true >> ${LOG_FILE}
```

Comment:
This breaks the redirection, you must write 
systemctl restart rudder-jetty >> ${LOG_FILE} || true